SimExLite/PropagationCalculators/WPGProgagationCalculator.py

```diff
@@ -0,0 +1,112 @@
+""":module WPGPropagationCalculator: Module that holds the WPGPropagationCalculator class."""
+
+from pathlib import Path
+import shutil
+import sys
+
+# Requires WPG package: https://github.com/samoylv/WPG
+from s2e.prop import propagate_s2e
+
+from libpyvinyl import BaseCalculator, CalculatorParameters
+from libpyvinyl.BaseData import DataCollection
+from SimExLite.WavefrontData import WavefrontData, WPGFormat
+
+
+class WPGPropagationCalculator(BaseCalculator):
+    """:class WPGPropagationCalculator: Class representing photon propagation through X-ray optics."""
+
+    def __init__(
+        self,
```

Comment:
docstring missing: document all parameters with 
```
:param <name>: <explanation
:type  <name>: <type>
```
or use type hints instead of type docstrings